### PR TITLE
Spark recommenders example

### PR DESCRIPTION
**Pull request by mwitiderrick**

### CODE REVIEW

File: spark-recommender-system/.layer/project.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Project configuration file
+# For more information on Project Configuration: https://docs.beta.layer.co/docs/projects/project-yml
+
+apiVersion: 1
+
+# Name of your project
+name: "Spark recommender system"
```
Comment (mecevit) [inline spark-recommender-system/.layer/project.yaml:7]: Lets fix name for the new projects: `spark-recommender-system`

File: spark-recommender-system/README.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+training_size = 0.8
+random_state = 0
+test_size = 0.2
+training, testing = data.randomSplit([training_size, test_size], seed=random_state)
+test_user = testing.filter(testing['USERID']==23).select(['MOVIEID','USERID'])
+spark_recommender_system = layer.get_model("spark_recommender_system")
+spark_recommender = spark_recommender_system.get_train()
+test_user_recommendations = spark_recommender.transform(test_user)
+test_user_recommendations.orderBy('prediction',ascending=False).show()
+```
```
Comment (mecevit) [inline spark-recommender-system/README.md:324]: Let's show the recommended movies here

File: spark-recommender-system/spark_model.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+"""New Project Example
```
Comment (mecevit) [inline spark-recommender-system/spark_model.py:1]: `Spark Recommender Project`

**Review by mecevit — CHANGES_REQUESTED** [review]

File: spark-recommender-system/.layer/project.yaml (hunk 1 above)
Comment (mwitiderrick) [reply spark-recommender-system/.layer/project.yaml:7]: @mecevit what do you mean here?

Comment (mecevit) [reply spark-recommender-system/.layer/project.yaml:7]: https://github.com/The-Layer/monorepo/pull/4500/files#diff-da2cccb7d7da20ad5c85532005ecf569f32e835f705825f551185571470e488fR12-R13

File: spark-recommender-system/README.md (hunk 2 above)
Comment (mwitiderrick) [reply spark-recommender-system/README.md:324]: on it

**Review by mecevit — APPROVED** [review]

**Pull request merged**